### ISSUE TITLE
Documentation for xrSetInputDeviceStateXxxxEXT APIs is incorrect/copied from first API

### ISSUE THREAD

**Issue by joejo-unity** [KhronosGroup/OpenXR-Docs#60]
Doc location: https://www.khronos.org/registry/OpenXR/specs/1.0/html/xrspec.html#XR_EXT_conformance_automation

First API has correct documentation for fourth parameter: **xrSetInputDeviceActiveEXT**
```

Parameter Descriptions

    session is the XrSession to set the input device state in.

    interactionProfile is the path representing the interaction profile of the input device (e.g. /interaction_profiles/khr/simple_controller).

    topLevelPath is the path representing the input device (e.g. /user/hand/left).

    isActive is the requested activation state of the input device. <--- This Parameter here

```

The rest of these seem to have just copied the same parameter documentation and all show the same `isActive` documentation for last parameter:

**xrSetInputDeviceStateBoolEXT**
**xrSetInputDeviceStateFloatEXT**
**xrSetInputDeviceStateVector2fEXT**
**xrSetInputDeviceLocationEXT**

**Comment by rpavlik** [comment]
Fix is awaiting merge. Thanks for the report!

(You shouldn't be using the extension, though, unless you're building a runtime in addition to a game engine! :D )

**Comment by rpavlik** [comment]
Resolved in 1.0.15. Thanks for your report!
